src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainSpecBasedSpecProvider.cs

```diff
@@ -247,6 +269,18 @@ private ReleaseSpec CreateReleaseSpec(ChainSpec chainSpec, long releaseStartBloc
             return releaseSpec;
         }
 
+        private (IReleaseSpec?, IReleaseSpec?) GetCurrentAndPreviousFork(ChainSpec chainSpec, ulong? releaseStartTimestamp = null)
```

Comment:
Prague.Instance and Cancun.Instance used to be use for tests only, we need to think if we can do it better, but out of the scope of this PR

Reply:
i think this is the issues with the way we are doing blob schedule using fork names.

Reply:
yeah, that's ok for now